src/components/CodeEditor/index.jsx

```diff
@@ -0,0 +1,67 @@
+import { Component } from 'react';
+import { object, string, func, number } from 'prop-types';
+import CodeMirror from '@skidding/react-codemirror';
+import { withStyles } from 'material-ui/styles';
+import 'codemirror/mode/javascript/javascript';
+import 'codemirror/mode/yaml/yaml';
+import 'codemirror/addon/display/placeholder';
+import 'codemirror/lib/codemirror.css';
+import 'codemirror/addon/lint/lint.css';
+import 'codemirror/theme/material.css';
+import './yaml-lint';
+import './styles.css';
+
+@withStyles({
+  codeMirror: {
```

Comment:
Add global styles with:

```js
@withStyles({
  '@global .CodeMirror': {
    font-size: 13,
    height: '100%',
  },
})
```

Reply:
That doesn't seem to work even after changing `font-size` to `fontSize`.

Reply:
Hmm weird, this should have worked.